src/js/components/productSupplier/create/subsections/PackageSpecification.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useEffect } from 'react';
+
+import PropTypes from 'prop-types';
+import { Controller } from 'react-hook-form';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { fetchQuantityUoM } from 'actions';
+import SelectField from 'components/form-elements/v2/SelectField';
+import TextInput from 'components/form-elements/v2/TextInput';
+import Subsection from 'components/Layout/v2/Subsection';
+
+const PackageSpecification = ({ control, errors }) => {
+  const dispatch = useDispatch();
+
+  const {
+    quantityUoM,
+  } = useSelector(state => ({
+    quantityUoM: state.unitOfMeasure.quantity,
+  }));
+
+  useEffect(() => {
+    dispatch(fetchQuantityUoM());
+  }, []);
+
+  return (
+    <Subsection
+      title={{
+        label: 'react.productSupplier.form.subsection.packageSpecification',
+        defaultMessage: 'Package Specification',
+      }}
+      collapsable={false}
+    >
+      <div className="form-grid-3">
+        <Controller
+          name="uomCode"
+          control={control}
+          render={({ field }) => (
+            <SelectField
+              {...field}
+              required
+              title={{
+                id: 'react.productSupplier.form.uomCode.title',
+                defaultMessage: 'Default Source Package',
+              }}
+              tooltip={{
+                id: 'react.productSupplier.form.uomCode.tooltip',
+                defaultMessage: 'The most common package purchased for this product',
+              }}
+              options={quantityUoM}
+              errorMessage={errors.uomCode?.message}
+            />
+          )}
+        />
+        <Controller
+          name="packageSize"
+          control={control}
+          render={({ field }) => (
+            <TextInput
+              {...field}
+              required
+              type="number"
+              errorMessage={errors.packageSize?.message}
+              title={{
+                id: 'react.productSupplier.form.packageSize.title',
+                defaultMessage: 'Package Size',
+              }}
+              tooltip={{
+                id: 'react.productSupplier.form.packageSize.tooltip',
+                defaultMessage: 'The number of units per package',
+              }}
+            />
+          )}
+        />
+        <Controller
+          name="minimumOrderQuantity"
```

Comment:
let's name it s `minOrderQuantity`, as the field on the `ProductSupplier` domain is named like this.